Dockerfile

```diff
@@ -29,12 +29,16 @@ WORKDIR /opt/iotaul
 RUN \
```

Comment:
Mabye we can include in this PR the same .dockerignore file as in https://github.com/telefonicaid/iotagent-json/pull/344. What do you think?

Reply:
Actually, it seems has been already done in a separate PR: https://github.com/telefonicaid/iotagent-ul/pull/305

Thus, NTC with regards this PR